core/src/main/resources/reference.conf

```diff
@@ -1369,6 +1369,30 @@ datastax-java-driver {
 
       shutdown {quiet-period = 2, timeout = 15, unit = SECONDS}
     }
+    # The timer used for scheduling request timeouts and speculative executions
+    timer {
+      # The timer tick duration
+      # This is the how frequent the timer should wake-up to check for timed out tasks.
+      # Lower resolution (i.e. longer durations) will leave more CPU cycles for running I/O
+      # operations at the cost of precision of exactly when a request timeout will expire or a
+      # speculative execution will run. Higher resolution (i.e. shorter durations) will result in
+      # more precise request timeouts and speculative execution scheduling, but at the cost of CPU
+      # cycles taken from I/O operations, which could lead to lower overall I/O throughput.
+      #
+      # Required: yes
+      # Modifiable at runtime: no
+      # Overridable in a profile: no
+      tick-duration = 100 milliseconds
+
+      # Number of ticks in a Timer wheel. The underlying implementation uses Netty's
+      # HashedWheelTimer, which uses a hashes to arrange the timeouts. This effectively controls the
+      # size of the timer wheel.
```

Comment:
👍 great docs.

Nit: I think there is a word missing in "a hashes to arrange the timeouts".

Reply:
Good catch, I think I originally meant it to say "a hash to arrange the timeouts" and then decided "hashes to arrange the timeouts" and forgot to remove the "a".